app/util/appUtil.js

```diff
@@ -114,6 +114,12 @@ async function getLastTokenId() {
   return lastTokenId ? parseInt(lastTokenId, 10) : 0
 }
 
+async function getMembers() {
+  await api.isReady
+
+  return api.query.membership.members()
```

Comment:
What does this return? Is it just an array of hex strings? If so we might want to wrap these into objects so the output is like:

```json
[
  {
    "address": "0x054535..."
  },
  {
    "address": "0x4355654..."
  }
]
```

That will allows us later to add more information arround these accounts without breaking anything

Reply:
Now returning a list of objects. However, returning res.body
`{ members: [ address: 'fooOne', address: 'fooTwo] }` 

instead of desired:
`{ [ address: 'fooOne', address: 'fooTwo] }` 

having issues removing the members property because of how the typing is returned

Reply:
think I got it now...